content/docs/start/model-management/model-cicd.md

```diff
@@ -191,8 +284,12 @@ DVC in the CICD workflow. You will also need to checkout the repository in the
 workflow. You can see more details in the
 [documentation](/doc/command-reference/artifacts/get#description).
 
-You can now use the following template to create your own Model Registry CICD
-actions on GitHub!
+You can now use the following templates to create your own Model Registry CICD
```

Comment:
This might also be better in its own section/conclusion for easy reference and to separate it from the step-by-step breakdown above.